tests/console/php_version.pm

```diff
@@ -22,24 +22,17 @@ use registration qw(add_suseconnect_product remove_suseconnect_product get_addon
 sub run {
     select_serial_terminal;
 
-    add_suseconnect_product(get_addon_fullname('script'), (is_sle('<15') ? '12' : undef)) if (is_sle('<15-sp4'));
 
-    # On sles15sp5, due to bus#1204824, the package "sle-module-web-scripting-release" is pre-installed,
-    # which tricks SUSEConnect to assume that this module is activated which it isn't, so add below
-    # workaround
-    if (is_sle('=15-SP5')) {
+    if (is_sle) {
+        # Check if BSC#1204824 is present
         if (script_run("suseconnect -l | grep 'Web and Scripting Module'| grep '(Activated)'") == 0) {
-            if (script_run('zypper se php8') == 104) {
-                record_soft_failure 'bsc#1204824 -  php8 packages are not available on sles15sp5';
-                remove_suseconnect_product('sle-module-web-scripting');
-                add_suseconnect_product('sle-module-web-scripting');
-            }
-        }
-        else {
-            add_suseconnect_product('sle-module-web-scripting');
+            die 'bsc#1204824 - Module activated already';
         }
+        add_suseconnect_product(get_addon_fullname('script'));
```

Comment:
Can you run the test also on other versions ? "script" module was added before now after

Reply:
done

Reply:
Thx, btw all the adding of modules on maintenance tests should be not done, all modules should be already there. But I guess it's from the previous 15-SP5 in development